package.json

```diff
@@ -11,7 +12,7 @@
       "require": "./dist/@lookingglass/webxr.umd.cjs"
     }
   },
-  "version": "0.3.1",
+  "version": "0.4.0-alpha.3",
```

Comment:
before being merged, this would need to be changed to 0.4.0

Reply:
fixed here: https://github.com/Looking-Glass/looking-glass-webxr/pull/31/commits/958abd41b86a3c0c9b86db7921a837cd6965d72f